server/service/src/processors/transfer/shipment/common.rs

```diff
@@ -25,18 +28,25 @@ pub(crate) fn generate_inbound_shipment_lines(
                  location_id: _,
                  batch,
                  expiry_date,
-                 pack_size,
+                 mut pack_size,
                  cost_price_per_pack: _,
                  sell_price_per_pack,
-                 number_of_packs,
+                 mut number_of_packs,
                  note,
                  r#type,
                  total_after_tax: _,
                  total_before_tax: _,
                  tax,
                  inventory_adjustment_reason_id: _,
              }| {
-                let cost_price_per_pack = sell_price_per_pack;
```

Comment:
I think it's a good idea to be `dry` when logic is split between context (i.e. define pack to one conversion in one place)

```rust
  let lines_iter = outbound_lines.into_iter().map(|l| l.invoice_line_row);

    let lines_iter = match store_preferences.pack_to_one {
        true => lines_iter.map(convert_invoice_line_to_single_pack),
        false => lines_iter,
    };

    let inbound_lines = lines_iter
        .map(
            |InvoiceLineRow {
```

Reply:
[6e56076](https://github.com/openmsupply/open-msupply/pull/1167/commits/6e56076c3c08b3a142d5510183baca26f7437df4)